### DISCUSSION TITLE
请问如何设置openai api key？

### DISCUSSION THREAD

**Discussion by godchenmeng** [mushan0x0/AI0x0.com#368] [Q&A] [answered]
我在设置里面设置了openai api key，但是在使用的时候总是提示
{
  "error": {
    "message": "The model: `gpt-4` does not exist",
    "type": "invalid_request_error",
    "param": null,
    "code": "model_not_found"
  }
}

**Comment by mushan0x0** [comment, marked as answer]
你选择的面具是4.0的面具，但是你的key不支持4.0，可以新开一个对话不使用面具
